data-serving/data-service/package.json

```diff
@@ -11,12 +11,13 @@
   "scripts": {
     "build": "tsc",
     "dev": "nodemon",
-    "import-data": "mongoimport --db dev --collection cases --file ../samples/line-list.json",
+    "import-data": "mongoimport --db covid19 --collection cases --file ../samples/line-list.json",
```

Comment:
how are we going to handle dev/prod instances down the road? obviously this is fine locally but when we're deploying to both prod and non-prod envs in AWS, will we name our dbs in Atlas something like covid19-dev and covid19-prod?

Reply:
We could keep the same db name between dev and prod, the "instances" will be different, that script currently connects to the default local DB and can't upload to atlas for example.

Or we could separate them yes and have covid19-dev || dev-covid-19, I have no strong preference.

Reply:
hmm... separate instances would probably be cleaner (don't have to change the db names across envs) -- maybe different sharded clusters for each? https://docs.atlas.mongodb.com/reference/glossary/#term-sharded-cluster

anyway, we can decide later once we start setting up CI/CD